pkg/flow/root_eval_context.go

```diff
@@ -1,76 +0,0 @@
-package flow
-
-import (
-	"github.com/grafana/agent/pkg/flow/internal/funcs"
-	"github.com/hashicorp/hcl/v2"
-	"github.com/zclconf/go-cty/cty/function"
-	"github.com/zclconf/go-cty/cty/function/stdlib"
-)
-
-// rootEvalContext is a set of global variables and functions which will be
-// available to all expressions in HCL.
-//
-// controller.Loader creates a child of this context which includes values for
-// running components.
-var rootEvalContext = &hcl.EvalContext{
```

Comment:
This has been removed in favor of the native River stdlib. River's stdlib is much smaller, but we can add to it over time as we identify functions we want to expose.